src/xrSound/OpenALDeviceList.cpp

```diff
@@ -199,8 +199,8 @@ void ALDeviceList::SelectBestDevice()
                 continue;
 
             GetDeviceVersion(i, &majorVersion, &minorVersion);
-            if (majorVersion > best_majorVersion ||
-                majorVersion == best_majorVersion && minorVersion > best_minorVersion)
+            if ((majorVersion > best_majorVersion) ||
+                ((majorVersion == best_majorVersion) && minorVersion) > best_minorVersion)
```

Comment:
Wrong parentheses.

Reply:
You mean that its should be like that?
```c++
if ((majorVersion > best_majorVersion) ||
    ->( ((majorVersion == best_majorVersion) && minorVersion) > best_minorVersion) )*<-*
```

Reply:
Clang is complaining about this?
If true, then is should be:
```cpp
if (majorVersion > best_majorVersion ||
	(majorVersion == best_majorVersion && minorVersion > best_minorVersion))
```
If no warnings, then we don't need to change the code at all.

Reply:
I have removed non-necessary brackets.